docs/docs/user-guide/using-cli-context.md

```diff
@@ -0,0 +1,107 @@
+# Use CLI for Microsoft 365 context
+
+CLI for Microsoft 365 context may store any kind of option and its value. 
+
+## How does it work
```

Comment:
This section seems to me more of a TL;DR section that you would include at the end.

Reply:
sure. moved to the bottom just before the example. or should this be also below the example chapter 🤔